### PR TITLE
Refactor test and change IP range

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR moves the test_data from the specific test to the central location - test_data.py and also changes the network cidr from 10.1.0.0/16 to 10.2.0.0/16 as it conflicts with the IP range of the lab where trillian jobs runs - thus leading to failures, which could be confusing

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
```
est Site 2 Site VPN Across redundant VPCs ... === TestName: test_01_redundant_vpc_site2site_vpn | Status : SUCCESS ===
ok
Test Site 2 Site VPN Across VPCs ... === TestName: test_01_vpc_site2site_vpn_multiple_options | Status : SUCCESS ===
ok
Test Remote Access VPN in VPC ... === TestName: test_01_vpc_remote_access_vpn | Status : SUCCESS ===
ok
Test Site 2 Site VPN Across VPCs ... === TestName: test_01_vpc_site2site_vpn | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 4 tests in 1812.057s

OK

```

### CODE REVIEW

**Review by weizhouapache — APPROVED** [review]
code lgtm

**Comment by Pearl1594** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
lgtm, but are we sure 4.16 is early enough for this? don´t we want this on ancient versions as well? at least 4.14/4.15?

**Comment by nvazquez** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]
LGTM - will be need to be placed on the 4.17 branch also and merge forward to main branch

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Comment by rohityadavcloud** [comment]
@Pearl1594 this needs fixing, smoketests are failing

@blueorangutan test

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Comment by shwstppr** [comment]
@Pearl1594 can you please check the failures

**Comment by Pearl1594** [comment]
Sorry for the delay @shwstppr. Will check the failures, however, it isn't a necessary fix for 4.17.1. Thanks.

**Comment by shwstppr** [comment]
@Pearl1594 thanks for the update. Moving to the next milestone for now.

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by sureshanaparti — APPROVED** [review]

**Comment by DaanHoogland** [comment]
that is not looking good :( applying insanity to make sure
@blueorangutan test

**Comment by DaanHoogland** [comment]
These errors seem to be consistent. Can you advise on how to proceed @Pearl1594 ?

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by alexandremattioli** [comment]
Could we change it to something like 10.129.0.0/16 please?  We have the range 10.0.0.0/10 (10.0.0.0-10.64.0.0) already pre-setup on firewalls, IPSEC, etc... for future labs. Doesn't need to be now though, but would be good.

File: tools/marvin/marvin/config/test_data.py [hunk 1]
```diff
@@ -1906,6 +1906,156 @@
...
+            "serviceCapabilityList": {
+                "SourceNat": {
+                    "RedundantRouter": 'true'
+                }
+            },
+        },
+        "vpc": {
+            "name": "TestVPC",
+            "displaytext": "TestVPC",
+            "cidr": '10.2.0.0/16'
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2001]:
```suggestion
            "cidr": '10.254.0.0/16'
```

File: tools/marvin/marvin/config/test_data.py [hunk 2]
```diff
@@ -1906,6 +1906,156 @@
...
+        },
+        "vpc1": {
+            "name": "TestVPC",
+            "displaytext": "VPC1",
+            "cidr": '10.2.0.0/16'
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2006]:
```suggestion
            "cidr": '10.254.0.0/16'
```

File: tools/marvin/marvin/config/test_data.py [hunk 3]
```diff
@@ -1906,6 +1906,156 @@
...
+        },
+        "vpc2": {
+            "name": "TestVPC",
+            "displaytext": "VPC2",
+            "cidr": '10.3.0.0/16'
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2011]:
```suggestion
            "cidr": '10.253.0.0/16'
```

File: tools/marvin/marvin/config/test_data.py [hunk 4]
```diff
@@ -1906,6 +1906,156 @@
...
+        },
+        "network_1": {
+            "name": "Test Network",
+            "displaytext": "Test Network",
+            "netmask": '255.255.255.0',
+            "gateway": "10.2.1.1"
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2017]:
```suggestion
            "gateway": "10.254.1.1"
```

File: tools/marvin/marvin/config/test_data.py [hunk 5]
```diff
@@ -1906,6 +1906,156 @@
...
+        },
+        "network_2": {
+            "name": "Test Network",
+            "displaytext": "Test Network",
+            "netmask": '255.255.255.0',
+            "gateway": "10.3.1.1"
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2023]:
```suggestion
            "gateway": "10.253.1.1"
```

File: tools/marvin/marvin/config/test_data.py [hunk 6]
```diff
@@ -1906,6 +1906,156 @@
...
+        },
+        "vpn": {
+            "vpn_user": "root",
+            "vpn_pass": "Md1sdc",
+            "vpn_pass_fail": "abc!123",  # too short
+            "iprange": "10.3.2.1-10.3.2.10",
```
Comment (DaanHoogland) [inline tools/marvin/marvin/config/test_data.py:2029]:
```suggestion
            "iprange": "10.253.2.1-10.253.2.10",
```

**Review by DaanHoogland — COMMENTED** [review]
@alexandremattioli this is what we agreed upon, right?

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by alexandremattioli** [comment]
LGTM

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
These errors no longer occur in other PR-smoke test runs, So I will move this to a later milestone for now.

**Comment by shwstppr** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
Tested along with Trillian changes, all tests in test_vpc_vpn.py passed

```
[root@pr6512-t6574-kvm-centos7-marvin marvin]# cat /marvin//MarvinLogs/test_vpc_vpn_F36HCG/results.txt 
Test Site 2 Site VPN Across redundant VPCs ... === TestName: test_01_redundant_vpc_site2site_vpn | Status : SUCCESS ===
ok
Test Site 2 Site VPN Across VPCs ... === TestName: test_01_vpc_site2site_vpn_multiple_options | Status : SUCCESS ===
ok
Test Remote Access VPN in VPC ... === TestName: test_01_vpc_remote_access_vpn | Status : SUCCESS ===
ok
Test Site 2 Site VPN Across VPCs ... === TestName: test_01_vpc_site2site_vpn | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 4 tests in 1276.985s

OK
```

File: tools/marvin/marvin/config/test_data.py [hunk 7]
```diff
@@ -1881,29 +1912,181 @@
                 "name": "windowsxdtemplate",
                 "passwordenabled": False,
                 "ostype": "Windows 8 (64-bit)"
-            },
-        },
+            }
+        }
 
+    "configurableData":
+        {
```
Comment (shwstppr) [inline tools/marvin/marvin/config/test_data.py:1919]: It seems this PR broke test_data.py https://github.com/apache/cloudstack/actions/runs/5186024194/jobs/9346611059#step:12:104 cc @Pearl1594 @weizhouapache @rohityadavcloud @DaanHoogland 

Failures were not seen earlier in the Trillian test runs as that uses its own test_data. I suggest we revert file and add only changes for `vpc_vpn` key ln 1942-2089?

**Pull request merged**